solidity/bin/check_link_balance

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+const {abi, wallet, web3} = require('../app/env.js')
+
+if (process.argv.length < 3) {
+  console.error('Usage: ./check_link_balance <token address> <holder address>')
+  process.exit(1)
+}
+
+const link = process.argv[2]
+const holder = process.argv[3]
+
+const funcSelector = '0x70a08231'// "balanceOf(address)"
+const encoded = abi.rawEncode(['address'], [holder])
+
+wallet.call({
```

Comment:
I believe we need `await` on all `wallet.call` and `wallet.send` methods since they're async.